### DISCUSSION TITLE
[v4 alpha1] Git Blame empty column with checkboxes off

### DISCUSSION THREAD

**Discussion by andrey-helldar** [gitextensions/gitextensions#10198] [Q&A] [answered]
### Environment

- GitExtensions version: 4.0.0.15025 (v4 alpha1)
- GIT version: 2.37.3.windows.1
- OS version: Windows 11 Pro (10.0.22000.0)
- .NET version: 6.0.9

### Issue description

I want to disable the display of the column with the authors of the changes, but when I do this, the window remains empty.

![image](https://user-images.githubusercontent.com/10347617/190896676-4c616444-47c0-457a-9dff-b579a1a5f038.png)

![image](https://user-images.githubusercontent.com/10347617/190896690-4cdff233-69ea-410d-94d8-efaf75e7e8f4.png)

### Steps to reproduce

- Tools > Settings
- Detailed > Blame Viewer
- I turn off all checkboxes in the results block:

### Did this work in previous version of GitExtensions?

Version 3.5.4 did not have this window. It appeared in 4 alpha.

**Comment by mstv** [comment, marked as answer]
> I want to disable the display of the column with the authors of the changes

In this case, just type `b` in order to toggle to the plain file view.
Alternatively, there is a split bar which you could move.

**Comment by andrey-helldar** [comment]
@mstv, moving the separator bar doesn't help much - when you move the mouse cursor, different lines are highlighted, which is a big hindrance.

![image](https://user-images.githubusercontent.com/10347617/190897898-abad2750-c8e1-45de-8306-374a75b24a2b.png)

PS: Hotkey helped. Thank you!

![image](https://user-images.githubusercontent.com/10347617/190897959-d6e2b3ca-b282-47b6-a086-03468482c356.png)
